Modules/Services/Sources/Services/Membership/Model/MembershipTierPaymentType.swift

```diff
@@ -0,0 +1,27 @@
+import StoreKit
+import ProtobufMessages
+
+
+public typealias MemberhipTierPaymentPeriodType = Anytype_Model_MembershipTierData.PeriodType
+
+public struct StripePaymentInfo: Hashable, Equatable {
+    public let periodType: MemberhipTierPaymentPeriodType
+    public let periodValue: UInt32
+    public let priceInCents: UInt32
+    
+    public var displayPrice: String {
+        return "\(priceInCents/100).\(priceInCents%100)"
```

Comment:
please use formatter for support different regions and different rules for currency.
 NumberFormatter.Style.currency

Reply:
Also please fix DateFormatter in SharedContentImporter